From 2edd5dbb9ad20a34acd58f8a46bb2a43c28815b7 Mon Sep 17 00:00:00 2001
From: Luca Weiss <luca@z3ntu.xyz>
Date: Tue, 15 Sep 2020 18:32:10 +0200
Subject: [PATCH] Add missing header
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

[...]/url-dispatcher/service/bad-url/exec-tool.cpp: In function ‘int main(int, char**)’:
[...]/url-dispatcher/service/bad-url/exec-tool.cpp:29:19: error: ‘runtime_error’ in namespace ‘std’ does not name a type
   29 |     } catch (std::runtime_error &e) {
      |                   ^~~~~~~~~~~~~
---
 service/bad-url/exec-tool.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/service/bad-url/exec-tool.cpp b/service/bad-url/exec-tool.cpp
index f3336cf..aea9ddb 100644
--- a/service/bad-url/exec-tool.cpp
+++ b/service/bad-url/exec-tool.cpp
@@ -19,6 +19,7 @@
 
 #include <ubuntu-app-launch/helper.h>
 #include <glib.h>
+#include <stdexcept>
 
 int
 main (int argc, char * argv[])
-- 
2.28.0

